packages/azure-themes/src/azure/AzureThemeDark.ts

```diff
@@ -5,55 +5,89 @@ import { FontSizes } from './AzureType';
+  buttonTextDisabled: 'rgb(121, 119, 117)',
   buttonTextHovered: DarkSemanticColors.secondaryButton.hover.border,
   buttonTextPressed: DarkSemanticColors.secondaryButton.pressed.border,
+  checkboxBorder: 'rgb(243, 242, 241)',
+  checkboxBorderHover: '',
+  checkboxBorderChecked: '',
+  checkmarkColorHover: 'rgb(161, 159, 157)',
+  checkboxBorder2: 'rgb(40, 153, 245)',
+  checkboxBorderDisabled: 'rgb(72, 70, 68)',
+  checkboxCheckedDisabled: 'rgb(72, 70, 68)',
+  checkboxCheckedMarkDisabled: '#000000',
+  checkboxIndeterminateBackground: 'transparent',
+  checkMarkFill: 'rgb(27, 26, 25)',
+  controlGroupBorder: 'rgb(243, 242, 241)',
+  checkBoxselectedBorderHover: 'rgb(108, 184, 246)',
+  groupChoiceFill: '#0078D4',
+  contextMenuBackground: '#000000',
   disabledBackground: CommonSemanticColors.backgrounds.disabled,
-  disabledBodyText: DarkSemanticColors.text.disabled,
+  disabledToggleBackground: 'transpanret',
+  disabledBodyText: 'rgb(121, 119, 117)',
+  disabledIconColor: 'rgb(72, 70, 68)',
+  dropdownBorder: 'rgb(161, 159, 157)',
   errorBackground: DarkSemanticColors.controlOutlines.error,
   errorText: DarkSemanticColors.text.error,
-  focusBorder: DarkSemanticColors.controlOutlines.accent,
+  focusBorder: '#4894fe',
+  dropdownBorderHover: '#0078D4',
+  iconButtonDisabledColor: '#A19F9D',
+  iconHoveredColor: '#106EBE',
+  iconPressedColor: 'rgb(108, 184, 246)',
   inputBackground: DarkSemanticColors.background,
   inputText: DarkSemanticColors.text.value,
   inputBorder: CommonSemanticColors.textControlOutline.rest,
   inputBorderHovered: CommonSemanticColors.textControlOutline.hover,
   inputPlaceholderText: DarkSemanticColors.text.placeholder,
-  link: DarkSemanticColors.text.hyperlink,
-  linkHovered: DarkSemanticColors.text.hyperlink,
+  link: '#0078D4',
+  linkHovered: '#004578',
   listBackground: DarkSemanticColors.background,
   listHeaderBackgroundPressed: DarkSemanticColors.item.hover,
   listItemBackgroundChecked: DarkSemanticColors.item.select,
   listItemBackgroundCheckedHovered: DarkSemanticColors.item.select,
   listItemBackgroundHovered: DarkSemanticColors.item.hover,
   listText: DarkSemanticColors.text.body,
-  menuItemBackgroundHovered: DarkSemanticColors.item.hover,
-  menuItemBackgroundPressed: DarkSemanticColors.item.select,
-  primaryButtonBackground: DarkSemanticColors.primaryButton.rest.background,
-  primaryButtonBackgroundDisabled: DarkSemanticColors.disabledButton.background,
-  primaryButtonBackgroundHovered: DarkSemanticColors.primaryButton.hover.background,
-  primaryButtonBackgroundPressed: DarkSemanticColors.primaryButton.pressed.background,
-  primaryButtonText: DarkSemanticColors.primaryButton.rest.text,
-  primaryButtonTextDisabled: DarkSemanticColors.disabledButton.text,
-  primaryButtonTextHovered: DarkSemanticColors.primaryButton.hover.text,
+  menuItemBackgroundHovered: '#373737',
+  menuItemBackgroundPressed: 'rgba(128,128,128,.6)',
```

Comment:
nit: should you use 0.6 instead of .6 as alpha value?

Reply:
Revised and used all hex values instead